tests/plugins/webp-uploads/image-edit-tests.php

```diff
@@ -36,8 +36,12 @@ public function it_should_backup_the_sources_structure_alongside_the_full_size()
 
 		$backup_sizes = get_post_meta( $attachment_id, '_wp_attachment_backup_sizes', true );
 
-		$this->assertNotEmpty( $backup_sizes );
 		$this->assertIsArray( $backup_sizes );
+		$this->assertArrayHasKey( 'full-orig', $backup_sizes );
+		$this->assertArrayHasKey( 'thumbnail-orig', $backup_sizes );
+		$this->assertArrayHasKey( 'medium-orig', $backup_sizes );
+		$this->assertArrayHasKey( 'medium_large-orig', $backup_sizes );
+		$this->assertArrayHasKey( 'large-orig', $backup_sizes );
```

Comment:
Without assertions like this, PHPStan complains about the `foreach()` loop below being passed `$backup_sizes` as an empty array.